cli/azd/cmd/pipeline.go

```diff
@@ -55,19 +56,18 @@ func (pc *pipelineConfigFlags) Bind(local *pflag.FlagSet, global *internal.Globa
 }
 
 func pipelineActions(root *actions.ActionDescriptor) *actions.ActionDescriptor {
-	infraCmd := &cobra.Command{
-		Use:   "pipeline",
-		Short: "Manage GitHub Actions or Azure Pipelines.",
-		//nolint:lll
-		Long: `Manage GitHub Actions or Azure Pipelines.
-
-The Azure Developer CLI template includes a GitHub Actions and an Azure Pipeline configuration file in the ` + output.WithBackticks(`.github/workflows`) + ` and ` + output.WithBackticks(`.azdo/pipelines`) + ` directories respectively. The configuration file deploys your app whenever code is pushed to the main branch.
-
-For more information, go to https://aka.ms/azure-dev/pipeline.`,
-	}
-
 	group := root.Add("pipeline", &actions.ActionDescriptorOptions{
-		Command: infraCmd,
+		Command: &cobra.Command{
+			Use:   "pipeline",
+			Short: i18nGetText(i18nCmdPipelineShort),
+		},
+		HelpOptions: actions.ActionHelpOptions{
+			Description: getCmdPipelineHelpDescription,
+			Footer:      getCmdPipelineHelpFooter,
+		},
+		GroupingOptions: actions.CommandGroupOptions{
+			RootLevelHelp: actions.CmdGroupMonitor,
+		},
```

Comment:
The grouping option feels like it should be automatic based on the the grouping that already exists within the action descriptors / cobra commands.

Reply:
It is a different grouping logic...

Cobra groups define `top level` commands and the sub-commands for each of them.
The `RootLevelHelp` defines a class for each of the `top level` commands only. Cobra has no information about this grouping, and the group is defined and assigned for each command by `UX designer`.

The current 4 groups are:
-  "Configure and develop your app"
-  "Manage Azure resources and app deployments"
-  "Monitor, test and release your app"
-  "About, help and upgrade"

Reply:
The `GroupingOptions` from the action descriptor options is meant to be used for any future requirement of creating this type of `virtual grouping`.  
We might, for example, use different groups for different audiences. 
Or we could also create an output of commands which require `azd init` before they run and which don't. 
etc etc

Reply:
OK - makes sense that it is more of a logical group... I just don't think we should expose any custom cases in the cobra builder for this.  Setting the options are great and as long as we can use them in a dynamic manner we should be OK.